core/application/impl/app_configuration_impl.cpp

```diff
@@ -188,11 +189,30 @@ namespace kagome::application {
         return false;
       }
       target.emplace_back(std::move(ma_res.value()));
-      return true;
     }
     return not target.empty();
   }
 
+  bool AppConfigurationImpl::load_telemetry_uris(
+      const rapidjson::Value &val,
+      char const *name,
+      std::vector<telemetry::TelemetryEndpoint> &target) {
+    auto it = val.FindMember(name);
+    if (it != val.MemberEnd() and it->value.IsArray()) {
+      for (auto &v : it->value.GetArray()) {
+        if (v.IsString()) {
+          auto result = parseTelemetryEndpoint(v.GetString());
+          if (result.has_value()) {
+            target.emplace_back(std::move(result.value()));
+            continue;
+          }
+        }
+        return false;
```

Comment:
All the detailed logs would be produced inside of `AppConfigurationImpl::parseTelemetryEndpoint`, otherwise the execution would not reach that `return false` statement